app/views/coronavirus_landing_page/components/landing_page/_page_header.html.erb

```diff
@@ -24,30 +24,40 @@
         <div class="govuk-grid-column-two-thirds">
           <%= render "govuk_publishing_components/components/heading", {
             text: sanitize(header["title"]),
-            font_size: 19,
+            font_size: 24,
```

Comment:
Should this be changing in both if scenarios?

Reply:
Design is happy for this to go in as is, affecting the current design too, but equally happy to render them differently based on the existence of `header["list"]` or `header["description"]`

Reply:
Sure, so long as it's intentional.